Makefile

```diff
@@ -98,6 +98,10 @@ endif
 build: ensure-submodule-initialized grpc/buf build-module
 	cd viam-cartographer && cmake -Bbuild -G Ninja ${EXTRA_CMAKE_FLAGS} && cmake --build build
 
+build-asan: EXTRA_CMAKE_FLAGS += -DCMAKE_CXXFLAGS="-fno-omit-frame-pointer -fsanitize=address -fsanitize-address-use-after-scope -O1" -DCMAKE_EXE_LINKER_FLAGS="-fsanitize=address"
+build-asan: ASAN_OPTIONS="detect_leaks=1 detect_stack_use_after_return=true"
+build-asan: build-debug
```

Comment:
Is the same target meant to be listed 3 times?

Reply:
yes. This is how you change the options to a given target

Reply:
Feel free to resolve